opentreemap/treemap/css/sass/partials/_subheader.scss

```diff
@@ -31,14 +31,6 @@
         transition: .2s ease-in-out -webkit-transform;
         transition: .2s ease-in-out transform;
 
-        + .stats-bar {
```

Comment:
It was using a more specific selector and overriding the `> .stats-bar` selector below, so I merged the two sections together, keeping everything but the `position: absolute`